src/renderer/src/assets/style.css

```
@@ -833,13 +840,36 @@
   body {
-    /* background-color: var(--background); */
     color: var(--foreground);
     font-weight: var(--text-weight);
+    transition:
+      color var(--dc-motion-default) var(--dc-ease-out-soft),
+      background-color var(--dc-motion-default) var(--dc-ease-out-soft);
   }
 
   html {
     font-family: var(--dc-font-family);
+    text-rendering: optimizeLegibility;
```

Stylelint (17.9.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)